.wp-env.json

```diff
@@ -1,14 +1,14 @@
 {
 	"core": "WordPress/WordPress",
 	"plugins": [ "." ],
-	"themes": [ "./test/emptytheme" ],
+	"themes": [ "./test/emptytheme", "./test/themes/style-variations" ],
```

Comment:
I'm pretty sure this shouldn't be here. I added it so the tests pass for now until I can work out the correct entries for `tests.mappings`